TripPlaner/src/main/java/com/singapore/TripPlaner/Controller/CityController.java

```diff
@@ -0,0 +1,55 @@
+package com.singapore.TripPlaner.Controller;
+
+import com.singapore.TripPlaner.Model.City;
+import com.singapore.TripPlaner.Model.Persistent;
+import com.singapore.TripPlaner.Service.CityService;
+import com.singapore.TripPlaner.Service.dataacces.Reader;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.List;
+
+@Controller
+public class CityController {
+
+    private final CityService cityService;
+    private final Reader reader;
+
+    public CityController(CityService cityService, Reader reader) {
+        this.cityService = cityService;
+        this.reader = reader;
+    }
+
+
+
+    @GetMapping("/city")
```

Comment:
```suggestion
    @GetMapping("/cities")
```
Ten controller zwraca listę miast. W związku z tym użyjemy tu w endpoincie l. mn. rzeczownika city. I tak robimy w każdym przypadku z wyświetlaniem listy obiektów